js/binance.js

```diff
@@ -271,7 +296,12 @@ module.exports = class binance extends Exchange {
     }
 
     async fetchMarkets (params = {}) {
-        const response = await this.publicGetExchangeInfo (params);
+        const marketType = this.options['defaultMarket'];
+        let method = 'publicGetExchangeInfo';
+        if (marketType === 'futures') {
+            method = 'fapiPublicGetExchangeInfo';
+        }
```

Comment:
Maybe it lost `type` value in market info entry. 
In case of `spot` markets, it was seen as default value by ccxt. @kroitor 
But In case of binance futures, `type` value of market is "swap".

Reply:
Sample of lost items:
```python
{
...
 'spot': False,
 'futures': False,
 'type': "swap",
 'swap': True
...
}

```